app/controllers/reviews_controller.rb

```diff
@@ -17,6 +17,7 @@ def create
     @review.shoe = @shoe
     @review.user = current_user
     if @review.save
+      ReviewMailer.new_review(@review).deliver_now
```

Comment:
Premature optimization warning! In the long term, as a best practice, you may not want to feed the entire instance of `Shoe` into the mailer, and instead just feed it the `id`, and let it `.find` that instance again. It hits the database twice, sure, but the mailer itself will require less memory to instantiate. But this is fine for now.